sympy/integrals/tests/test_manual.py

```diff
@@ -600,6 +600,20 @@ def test_issue_23566():
     assert i == -log(4 - 2*sqrt(3)) + log(2)
     assert str(i.n()) == '1.31695789692482'
 
+def test_issue_25090():
```

Comment:
It should be `def test_issue_25093()`

Reply:
Whoops, thanks for pointing out, I've corrected it.